exonum/build.rs

```diff
@@ -77,16 +20,14 @@ fn main() {
     file.write_all(user_agent.as_bytes())
         .expect("Unable to write data to file");
 
-    protoc_generate(
-        &out_dir,
+    protobuf_generate(
         "src/encoding/protobuf/proto/",
```

Comment:
You can share this path to the other crates via `cargo:stuff=/your/path` and import them by the env variable `EXONUM_STUFF`.
Thus dependent crates will be able to use exonum proto files.

Reply:
I'll do this in the future PR for services.